scripts/pull_request_blaster_outer/lockdown_ruby_version.rb

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env ruby
+require 'yaml'
+
+if `grep -E 'git clone.+manageiq.git' bin/setup`.include?('manageiq.git')
```

Comment:
we should get what we're doing here into the filename... lockdown ruby versions for a manageiq release... naming is hard